app/api/v1/views.py

```diff
@@ -0,0 +1,83 @@
+from flask_restful import Api, Resource 
+from flask import jsonify, make_response, request
+import datetime
+
+from .models import RedFlagModel
+
+
+
+class RedFlags(Resource):
+    def __init__(self,CreatedBy, Location, Status, Comment):
+        self.db = RedFlagModel()
+         
+
+    def post(self): 
+        daatedBy = data['CreatedBy']
```

Comment:
Where are we using the properties you extracted from the request? You are passing `data` directly to `RedFlagModel()` - which required 4 parameters instead of one